src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Routes, Route } from 'react-router-dom';
+import './App.css';
+import BookList from './components/BookList';
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <>
+      <Navbar />
+      <main>
+        <Routes>
+          <Route path="/" element={<BookList />} />
+        </Routes>
+      </main>
+    </>
```

Comment:
- [ ] Kindly, It would be better to add  2 `<Route>`s  `/` and `/categories` in your App. Currently, the `categories` page is missing 😨; 
      
   please make sure that your App contains 2 pages components: `<Books>` and `<Categories>` 🙏